irc/client.py

```diff
@@ -918,6 +918,8 @@ def _handle_event(self, connection, event):
             matching_handlers = sorted(
                 self.handlers.get("all_events", []) + self.handlers.get(event.type, [])
             )
+            if len(matching_handlers) == 0 and event.type != "all_raw_messages" and event.type != "pong":
```

Comment:
It's starting to look like 'pong' and 'all_raw_messages' aren't special conditions. I wonder if there's a way that 'pong' could not be treated as a special condition (possibly by having a handler added by default). Can you add a comment explaining why "all_raw_messages" should be ignored?

Reply:
I originally used this downstream to only handle "really" unhandled events and `all_raw_messages` created duplicate events of things that were already handled as their non-raw versions.

Technically I suppose this could be moved to implementation of the handler to filter out events it doesn't want to handle so it's up to the user.

Pong is special because it is handled already so making the auto-pong an actual handler instead of doing it internally would probably be the right way.